package.json

```diff
@@ -12,6 +12,9 @@
   },
   "dependencies": {
     "@apollo/client": "3.3.21",
+    "@dnd-kit/core": "6.0.7",
+    "@dnd-kit/sortable": "7.0.2",
+    "@dnd-kit/utilities": "3.2.1",
```

Comment:
For anyone wondering, bundlephobia returns 13.9kB + 3.7kB + 1.6kB for these, pretty much the same as `react-dnd`.